server/src/main/java/org/elasticsearch/index/mapper/CompletionFieldMapper.java

```diff
@@ -402,7 +376,7 @@ public void parse(ParseContext context) throws IOException {
             }
             // truncate input
             if (input.length() > maxInputLength) {
-                int len = Math.min(maxInputLength, input.length());
```

Comment:
This was an IDE-recommended change, we've just asserted that maxInputLength is smaller than input.length(), so Math.min will always return it.